### DISCUSSION TITLE
node-datachannel v0.5.0

### DISCUSSION THREAD

**Discussion by nasselk** [geckosio/geckos.io#270] [General]
Hey, can you please upgrade to node-datachannel 0.5.0 ?

It seems to have multiple improvements, so it would be cool to have it in geckos

**Comment by yandeu** [comment]
I have just updated the lib. Hopefully the tests will pass.

Update:
It failed. Feel free to make a PR.

**Reply by nasselk** [reply]
Idk if it changes anything, but there's no support for nodejs under version 16

Node version <16 support dropped
N-API bindings/binaries (version 8)
WebRTC polyfills first phase
electron support and demo project
libdatachannel updated to latest version
Log messages added
